requirements.txt

```diff
@@ -0,0 +1 @@
+django-cms>=3.2
```

Comment:
Move this to `install_requires` in `setup.py` (like with the other plugin repos), and add a pin for <3.4